### ISSUE TITLE
KODI plugin for hyperion configuration is here :)

### ISSUE THREAD

**Issue by tociek** [tvdzwan/hyperion#358]
Hi All,

I just wanted to let you know that I wrote small plugin for KODI, that can help creating the config file for users that do not want to handle all advanced options available for hyperion. Basically, the only parameters that user provides is number of leds in each direction and where the strip begins.
It also adds grabber-v4l2 section to the config file, if grabber is detected.

I have tested in on OE and it looks like it works :) I would appreciate any feedback and testing (especially for clockwise setup and other systems than OE).

Here is the repository:
https://github.com/LightberryEu/plugin.program.hyperion.configurator

I hope this will be useful.

Jacek

**Comment by galleon111** [comment]
Hi, the configurator created a hyperion.config.json file alright, however, the contents of the file is mixed up. All the correct lines of code are there, but, in the wrong order. Ie the number of led are at the start, the   end of json, is in the middle, it's all mixed up.

**Comment by tociek** [comment]
Indeed sections are mixed up - it is because I am not able to force correct order via orderedDict in python. Despite that, config works, right?

**Comment by galleon111** [comment]
It sometimes works, there are random, stop, starts and flashing with the leds.




Sent from Samsung tablet

-------- Original message --------

**Comment by tociek** [comment]
Do you have other config file that works well with the same hardware setup?

**Comment by galleon111** [comment]
Yes, I returned to my usual config file and all is good.
The problem I was trying to fix was, the black borders on letterbox wide movies were not being detected.
I fixed it by commenting out the XBMC framegrabber in the config file.
I doen't use XBMC for anything, I have a lightberry hdmi premium kit with a 3d blu-ray player attached.
Works great with the lightberry kit. 65" samsung curved screen & 62 led, 3D movies are brilliant
Raspberry pi B+ , openelec, hyperion. 





Sent from Samsung tablet

-------- Original message --------
